src/realm.js

```diff
@@ -173,6 +174,7 @@ export class Realm {
   timeout: void | number;
   mathRandomGenerator: void | (() => number);
   strictlyMonotonicDateNow: boolean;
```

Comment:
Can you also wire this up to the command-line?

Reply:
Yup I've started on that already